gp-includes/routes/profile.php

```diff
@@ -40,7 +40,7 @@ public function profile_view( $user = '' ) {
 			$user = get_user_by( 'slug', $user );
 		}
 
-		if ( ! $user ) {
+		if ( ! $user || ( is_object( $user ) && ! $user->exists() ) ) {
```

Comment:
Should check explicit for `WP_User`: `$user instanceof WP_User`

Reply:
Added.